vm/devices/get/get_protocol/src/dps_json.rs

```diff
@@ -155,8 +155,9 @@ open_enum! {
 pub struct ManagementVtlFeatures {
     pub strict_encryption_policy: bool,
     pub _reserved1: bool,
```

Comment:
nit: add a comment:

/// reserved for future use for other encryption-related config, bits are semantically grouped.